modules/subnet/variables.tf

```diff
@@ -49,3 +49,8 @@ variable "freeform_tags" {
     module      = "oracle-terraform-modules/vcn/oci"
   }
 }
+
+variable "tenancy_id" {
+  description = "Tenancy OCID"
+  type        = string
```

Comment:
you can leave internal modules variables without description. see https://github.com/oracle-terraform-modules/terraform-oci-oke/blob/main/modules/iam/variables.tf